docs/apps/datalad.md

```diff
@@ -18,6 +12,12 @@ on this documentation page will refer to it.
 
 ## *DataLad* and *Docker*
 
+Apps may be able to identify if the input dataset is handled with
+*DataLad* or *Git-Annex*, and pull down linked data that has not
+been fetched yet.
```

Comment:
If user knows that data is fetched already, or data is to be fetched by an external execution wrapper (e.g. `datalad containers-run` or `datalad run` or `reproman run` used by openneuro), any problem with using datalad/git-annex could then be avoided by utilizing `--no-datalad-get` for mriqc.  It was also our with @asmacdo workaround for the 

- https://github.com/nipreps/mriqc/issues/1307

That is why we even added it as a default to our repronim/containers collection configuration for `datalad containers-run`: https://github.com/ReproNim/containers/pull/133/files  

I think it might be worth noting somewhere here in these docs (do not see mentioning of `--no-datalad-get` anywhere in this repo ATM).

Reply:
Thanks, I think more info about this would be important. I'll add a terse note here and a more complete description in MRIQC's documentation.

Reply:
Added in 44360cb. Please send further suggestions or reopen this conversation if that fell short of addressing the issue.